velox/docs/functions/presto/string.rst

```diff
@@ -134,13 +134,15 @@ String Functions
 
     Returns the rest of ``string`` from the starting position ``start``.
     Positions start with ``1``. A negative starting position is interpreted
-    as being relative to the end of the string.
+    as being relative to the end of the string. Return empty When the negative
```

Comment:
typo: When -> when

Perhaps, rephrase for clarity:

Returns empty string if absolute value of 'start' is greater then length of the `string`.